q2cli/handlers.py

```diff
@@ -310,12 +310,25 @@ def get_click_options(self):
 
     def get_value(self, arguments, fallback=None):
         import qiime2
-
+        import qiime2.sdk.util
         path = self._locate_value(arguments, fallback)
+
         if path is None:
             return None
+
+        try:
+            # check to see if path is an artifact
+            art = qiime2.Artifact.load(path)
```

Comment:
@jairideout we couldn't come up with a better way to check this. Should we create an API, or is this fine for now?